File: templates/tpl.cs
using System;
using System.IO;
using System.Text;

class Dropper
{{
    static void Main()
    {{
        // Array of base64 chunks
        string[] chunks = {{
{chunks}
        }};

        // Efficiently build the full base64 string
        StringBuilder b64 = new StringBuilder();
        foreach (string chunk in chunks)
        {{
            b64.Append(chunk);
        }}

        // Convert base64 string to byte array
        byte[] payload = Convert.FromBase64String(b64.ToString());
        
        // Write byte array to file
        File.WriteAllBytes("{filename}", payload);
        
        Console.WriteLine($"Wrote {{payload.Length}} bytes to {filename}");
    }}
}}

File: templates/tpl.java

import java.util.Base64;
import java.nio.file.Files;
import java.nio.file.Paths;
import java.io.IOException;

public class tpl {
    public static void main(String[] args) throws IOException {
        // Array of base64 chunks
        String[] chunks = {
                "{chunks}"
        };

        // Reconstruct full base64 string efficiently
        StringBuilder b64 = new StringBuilder();
        for (String chunk : chunks) {
            b64.append(chunk);
        }

        // Decode base64 to bytes
        byte[] payload = Base64.getDecoder().decode(b64.toString());

        // Write bytes to target file
        Files.write(Paths.get("{filename}"), payload);

        System.out.println("Wrote " + payload.length + " bytes to {filename}");
    }
}
